python/pipeline/experiment.py

```diff
@@ -170,6 +172,60 @@ class Software(dj.Lookup):
         ('imager', '1.0')]
 
 @schema
+class LaserCalibration(dj.Manual):
+    definition = """
+    # stores measured values from the laser power calibration
+
+    calibration_id      : int
+    ---
+    -> Person
```

Comment:
I would add an explicit `calibration_date` attribute for those cases when the calibration date differs from the time that the data are entered in the database.